Carmunication/Utils/FirebaseManager.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  FirebaseManager.swift
+//  Carmunication
+//
+//  Created by 김동현 on 2023/10/19.
+//
+
+import UIKit
+
+import FirebaseDatabase
+import FirebaseStorage
+
+class FirebaseManager {
```

Comment:
파일 생성하자마자 작성해줘야 까먹지 않더라고요 😅

```suggestion
final class FirebaseManager {
```